keyboards/adalyn/keymaps/vial/vial.json

```diff
@@ -0,0 +1,154 @@
+
```

Comment:
VIAL is not supported in the main repo, and the vial config and keymap should be removed.